### PR TITLE
Insecure connection by default

### PR DESCRIPTION
**Pull request by smacker**
Fix: #52

Python already open insecure connection by default: https://github.com/src-d/lookout-sdk/blob/master/python/lookout/sdk/grpc.py#L13

Go updated.

Doc for python `add_insecure_port` was placed in a wrong place in README (it has nothing to do with data server)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,14 +27,14 @@ For the gRPC client and server please follow these requirements:
 - support [RFC 3986 URI scheme](https://github.com/grpc/grpc-go/issues/1911); lookout-sdk includes helpers for this:
   - go: using `pb.ToGoGrpcAddress` and `pb.Listen`.
   - python: using `lookout.sdk.grpc.to_grpc_address`.
+- use insecure connection:
+  - currently lookout expects to use insecure gRPC connections
```
Comment (dpordomingo) [inline README.md]: Let people now that it's provided by `pb.DialContext` helper (as done with gRPC message size)
```suggestion
  - currently lookout expects to use insecure gRPC connections, as provide by `pb.DialContext`
```

Comment (carlosms) [reply README.md]: sorry to nitpick, typo in @dpordomingo's suggestion: as **provided** by

Comment (dpordomingo) [reply README.md]: :point_up: many thanks @carlosms 
FIXED :dagger:

Comment (smacker) [reply README.md]: fixed

**Pull request merged**